pkg/api/v1/router_server_components_test.go

```diff
@@ -517,17 +517,41 @@ func TestIntegrationServerUpdateComponents(t *testing.T) {
 		return err
 	})
 
-	componentSliceFixture := func() []serverservice.ServerComponent {
-		s := serverservice.ServerComponentSlice{}
+	// fixtures given to test cases below
+	var componentFixture []serverservice.ServerComponent
 
-		for _, c := range servers[0].Components {
+	var serverFixture serverservice.Server
+
+	// The component fixture targeted in test cases below
+	fixtureComponentName := "My Lucky Fin"
+	fixtureComponentVendor := "Barracuda"
+	fixtureComponentSerial := "Right"
+
+	// identify component and server fixture for test
+	for _, server := range servers {
+		server := server
```

Comment:
why do you copy the value that's already copied by the for loop? Do you need the copied value or do you need a reference to the item in the list?

Reply:
@JAORMX In this case we're keeping a copy of the server component, and so the `server := server` copy is unnecessary - most likely left over from a previous change, fixed in 81e7ea5.